presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusConfig.java

```diff
@@ -63,4 +64,17 @@ public PrometheusConfig setMaxQueryRangeDuration(String maxQueryRangeDuration)
         this.maxQueryRangeDuration = maxQueryRangeDuration;
         return this;
     }
+
+    @NotNull
+    public String getCacheConfigDuration()
+    {
+        return cacheConfigDuration;
+    }
+
+    @Config("cache-config-duration")
```

Comment:
I feel like this should just be `cache-duration` (across the board)

Reply:
Done in 4f44f28

Reply:
Done in 4f44f28e